service-manual/user-centred-design/resources/patterns/check-your-answers-pages.md

```diff
@@ -37,7 +37,7 @@ to drop out of the transaction before completing it
 
 # Code
```

Comment:
Sorry, I misread what this PR was doing.

IMHO we should remove the section entirely. The code in the kit is not production ready, and we should not be linking to it all.